### ISSUE TITLE
Lizenz-Wirr-Warr

### ISSUE THREAD

**Issue by tobwen** [CodeforRuhrgebiet/schulradar#2]
# Lizenz-Wirr-Warr
Bitte arbeitet ein wenig sorgfältiger, wenn ihr mit Datensätzen unterschiedlicher Herkunft arbeitet

## OpenGeoGB
OpenGeoGB tritt zwar nach außen als "gemeinfrei" auf, ist es aber nicht wirklich.
Die Änderungen der Postleitzahlen entstammen großteils den Änderungsmitteilungen der Deutschen Post AG. Diese hat sich wiederholt gegen eine Freigabe oder Verwendung ausgesprochen, da der Vertrieb der Postleitzahlen und verwander Produkte eine profitable Einnahmequelle für das Unternehmen ist.

## Google Maps Koordinaten
In https://github.com/CodeforRuhrgebiet/schulradar/blob/gh-pages/supported_cities.yml werden Daten des Angebots http://www.gpskoordinaten.de/ übernommen. Die Daten entstammen jedoch dem Dienst Google Maps. Es findet weder eine Kennzeichnung statt, noch erlaubt Google die öffentliche Verbreitung und Nutzungs der Geokodierungsergebnisse.

## LICENSE 
https://github.com/CodeforRuhrgebiet/schulradar/blob/gh-pages/LICENSE weißt nicht auf die unterschiedlichen Lizenzen in diesem Repository hin. Wir haben hier drei oder vier verschiedene Datenquellen, teilweise OpenData, teilweise nicht.

**Comment by gitviola** [comment]
Hi Tobias, danke für die Klarstellung. Ich werde versuchen das so schnell wie möglich zu beheben.

Bin gerade auf https://www.suche-postleitzahl.org/downloads gestoßen, was OpenGeoGB vollständig umgehen lässt. Und die Koordinaten lassen sich auch über OSM herausbekommen.


Da du dich anscheint sehr dafür interessierst und ich gesehen habe, dass du aus der Nähe kommst: komm doch einfach mal bei einem von unseren Treffen vorbei und hilf uns, dass es kein Lizenz-Wirr-Warr mehr gibt! :-)

**Comment by tobwen** [comment]
Ja, komme seit kurzem aus Essen und bin eigentlich in der Thematik recht firm (bin Geograph). Leider schaffe ich es heute Abend nicht (muss bis 19 Uhr arbeiten). Aber einige Mapper aus der OpenStreetMap-Community haben sich ja schon angekündigt.
